api/agentlocalpb/agentlocal.proto

```diff
@@ -24,6 +24,8 @@ message ServerInfo {
   google.protobuf.Duration latency = 5;
   // Clock drift from PMM Server (if agent is connected).
   google.protobuf.Duration clock_drift = 6;
+  // PMM Agent version (if agent is connected).
```

Comment:
agent_version should return always, not only when agent is connected, please fix the comment

Reply:
It is fixed, but due to comment from another PR it was also moved to StatusResponse.